### DISCUSSION TITLE
✅ Trace route in Tools

### DISCUSSION THREAD

**Discussion by tmcb82** [isontheline/pro.webssh.net#271] [Ideas]
I think it would be in incredibly help to have Trace route in the tools section.

**Comment by isontheline** [comment]
Hello @tmcb82 👋

You're right it would be incredible!

Issue created in order to track progress : https://github.com/isontheline/pro.webssh.net/issues/273

Have a nice day☀️

**Reply by isontheline** [reply]
A "simple" traceroute is now available inside the latest beta release : https://testflight.apple.com/join/QSrBK59z
